Description: Ignore the 'Unable to write uncompressed print data: Broken pipe'
 error that sometimes breaks the errorlines counting on kfreebsd for some
 reason
Origin: vendor
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2016-08-09

--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -984,7 +984,7 @@
 fi
 
 # Error log messages
-count=`$GREP '^E ' $BASE/log/error_log | $GREP -v '(usb) crashed on signal 11' | $GREP -v '(dnssd) stopped with status 1' | $GREP -v 'loadFile failed: temp file: not a PDF file' | $GREP -v 'Failed to connect to system bus' | $GREP -v -E 'Unable to open listen socket for address .* Address family not supported by protocol.' | wc -l | awk '{print $1}'`
+count=`$GREP '^E ' $BASE/log/error_log | $GREP -v '(usb) crashed on signal 11' | $GREP -v '(dnssd) stopped with status 1' | $GREP -v 'loadFile failed: temp file: not a PDF file' | $GREP -v 'Failed to connect to system bus' | $GREP -v -E 'Unable to open listen socket for address .* Address family not supported by protocol.' | $GREP -v 'Unable to write uncompressed print data: Broken pipe' | wc -l | awk '{print $1}'`
 if test $count != 33; then
 	echo "FAIL: $count error messages, expected 33."
 	$GREP '^E ' $BASE/log/error_log
